common/djangoapps/student/models.py

```diff
@@ -1338,7 +1338,7 @@ def update_enrollment(self, mode=None, is_active=None, skip_refund=False):
                 dog_stats_api.increment(
                     "common.student.enrollment",
                     tags=[u"org:{}".format(self.course_id.org),
-                          u"offering:{}".format(self.course_id.offering),
+                          u"offering:{}".format(self.course_id.run),
```

Comment:
A few questions here:

* Isn't offering effectively "course/run", not just the run?
* Do we really want to keep the "offering" label in the tag?
* How much do we still care about submitting information to DataDog?

Reply:
You're correct about offering - I wrongly assumed it was just the run.

I'm going to simply remove this DataDog submission. We don't use DataDog data for any formal auditing of course enrollments, so its removal won't break any critical systems. If someone is using the data, it'll be easy enough to add it back.